src/device/deletePendingInvitation.js

```diff
@@ -0,0 +1,21 @@
+const { default: API } = require('@aws-amplify/api-rest')
+
+const serializeAxiosResponse = require('../utils/serializeAxiosResponse')
+const { DEFAULT_REQ_OPTS } = require('../utils/defaultReqOpts')
+
+/**
+ * @desc retract user pending invitation
+ * @memberof nexus.device
+ * @function retract
+ * @async
+ * @param {Object} params
+ * @param {string} params.deviceId
+ * @param {Object} params
+ * @param {string} params.invitationId
+ * @returns {Promise<ApiResponse>}
+ */
+
+module.exports = serializeAxiosResponse(
+  ({ deviceId, invitationId }) => API
+    .del('device', `${deviceId}/user-invitation/${invitationId}`, { ...DEFAULT_REQ_OPTS }),
```

Comment:
The path is inconsistent and because hambda is not merged yet (https://github.com/ospin-web-dev/hambda/pull/2262), we should fix that: 

it should be `user-invitations` (plural). It is convention to use plural on collections